src/integrationTest/java/uk/gov/hmcts/reform/blobrouter/services/email/EmailSenderTest.java

```diff
@@ -0,0 +1,104 @@
+package uk.gov.hmcts.reform.blobrouter.services.email;
+
+import com.icegreen.greenmail.util.ServerSetupTest;
+import com.microsoft.applicationinsights.boot.dependencies.google.common.io.Resources;
+import org.apache.commons.mail.util.MimeMessageParser;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.api.extension.RegisterExtension;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import uk.gov.hmcts.reform.blobrouter.jupiter.GreenMailExtension;
+
+import java.io.File;
+import java.util.Map;
+import javax.activation.DataSource;
+import javax.mail.Address;
+
+import static java.util.Collections.emptyMap;
+import static org.assertj.core.api.Assertions.assertThat;
+
+@SpringBootTest
+@ExtendWith(MockitoExtension.class)
+public class EmailSenderTest {
+
+    // as defined in application.properties
+    private static final String USERNAME = "test_username";
+    private static final String PASSWORD = "test_password";
+
+    private static final String FROM_ADDRESS = "a@b.c";
+    private static final String SUBJECT = "subject";
+    private static final String RECIPIENT_1 = "Foo <foo@hmcts.net>";
+    private static final String RECIPIENT_2 = "bar@hmcts.net";
+    private static final String[] RECIPIENTS = {RECIPIENT_1, RECIPIENT_2};
+    private static final String FILE_NAME_1 = "email/test1.zip";
+    private static final String FILE_NAME_2 = "email/test2.zip";
+    private static final String BODY = "body";
+
+    @Autowired
+    private EmailSender emailSender;
```

Comment:
is it possible to not load full app with `@SpringBootTest` and just test this particular feature in integration way as it is in entire test class here? Loading up spring up seems unnecessary evil